<ui version="4.0" >
 <class>UserConfig</class>
 <widget class="QDialog" name="UserConfig" >
  <property name="windowModality" >
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>316</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="1" column="0" >
    <layout class="QFormLayout" name="formLayout" >
     <property name="fieldGrowthPolicy" >
      <enum>QFormLayout::FieldsStayAtSizeHint</enum>
     </property>
     <property name="margin" >
      <number>3</number>
     </property>
     <item row="0" column="1" >
      <widget class="QLineEdit" name="usernameInput" />
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Ability Level:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QComboBox" name="abilityLevelInput" />
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Weekly Practice:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="QComboBox" name="weekPracticeInput" />
     </item>
     <item row="3" column="0" >
      <widget class="QLabel" name="label_4" >
       <property name="text" >
        <string>Weekly Playing:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" >
      <widget class="QComboBox" name="weekPlayInput" />
     </item>
     <item row="4" column="0" >
      <widget class="QLabel" name="label_5" >
       <property name="text" >
        <string>Years Playing:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" >
      <widget class="QComboBox" name="yearsPlayingInput" />
     </item>
     <item row="5" column="0" >
      <widget class="QLabel" name="label_6" >
       <property name="text" >
        <string>Recording Metro:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1" >
      <widget class="QComboBox" name="recordingMetroInput" />
     </item>
     <item row="6" column="0" >
      <widget class="QLabel" name="label_7" >
       <property name="text" >
        <string>Playback Metro:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1" >
      <widget class="QComboBox" name="playbackMetroInput" />
     </item>
     <item row="0" column="0" >
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Username:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1" >
    <layout class="QFormLayout" name="formLayout_2" >
     <property name="fieldGrowthPolicy" >
      <enum>QFormLayout::FieldsStayAtSizeHint</enum>
     </property>
     <property name="margin" >
      <number>3</number>
     </property>
     <item row="6" column="0" >
      <widget class="QLabel" name="label_12" >
       <property name="text" >
        <string>Shortest note:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1" >
      <widget class="QDoubleSpinBox" name="noteWidthInput" >
       <property name="maximum" >
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep" >
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="5" column="0" >
      <widget class="QLabel" name="label_15" >
       <property name="text" >
        <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Lucida Grande'; font-size:13pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-weight:600; text-decoration: underline;">Intonation&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="indent" >
        <number>-1</number>
       </property>
       <property name="openExternalLinks" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0" >
      <widget class="QLabel" name="label_13" >
       <property name="text" >
        <string>Recording time:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QSpinBox" name="recordTimeInput" >
       <property name="minimum" >
        <number>10</number>
       </property>
       <property name="maximum" >
        <number>120</number>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="QDoubleSpinBox" name="onsetCutoffInput" >
       <property name="maximum" >
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep" >
        <double>0.010000000000000</double>
       </property>
       <property name="value" >
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_14" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text" >
        <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Lucida Grande'; font-size:13pt; font-weight:400; font-style:normal;">
&lt;p align="center" style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-weight:600; text-decoration: underline;">Rhythms&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="label_8" >
       <property name="text" >
        <string>Clap Volume:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" >
      <widget class="QLabel" name="label_9" >
       <property name="text" >
        <string>Align First Beat:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" >
      <widget class="QCheckBox" name="alignFirstBeatInput" >
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
     <item row="7" column="0" >
      <widget class="QLabel" name="label_16" >
       <property name="text" >
        <string>Just intonation:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1" >
      <widget class="QCheckBox" name="justInput" >
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1" >
    <widget class="QDialogButtonBox" name="dialogButtons" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="label_10" >
     <property name="text" >
      <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Lucida Grande'; font-size:13pt; font-weight:400; font-style:normal;">
&lt;p align="center" style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-weight:600; text-decoration: underline;">User Config&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QLabel" name="label_11" >
     <property name="text" >
      <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Lucida Grande'; font-size:13pt; font-weight:400; font-style:normal;">
&lt;p align="center" style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-weight:600; text-decoration: underline;">Audio Analysis Config&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>dialogButtons</sender>
   <signal>accepted()</signal>
   <receiver>UserConfig</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialogButtons</sender>
   <signal>rejected()</signal>
   <receiver>UserConfig</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
